infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -43,6 +43,8 @@ type TFBlueprintTest struct {
 	saKey                         string                   // optional setup sa key
 	tfDir                         string                   // directory containing Terraform configs
 	tfEnvVars                     map[string]string        // variables to pass to Terraform as environment variables prefixed with TF_VAR_
+	backendConfig                 map[string]interface{}   // backend configuration for terraform init
+	migrateState                  bool                     // suppress user confirmation in a migration in terraform init
```

Comment:
Would setting `TF_IN_AUTOMATION` by default help to suppress this?

Reply:
Hi @bharathkkb do you mean the flag [TF_IN_AUTOMATION](https://www.terraform.io/cli/config/environment-variables#tf_in_automation) ?

It does not suppress the error saying `If you wish to attempt automatic migration of the state, use "terraform init -migrate-state"`

One alternative would be to remove the `WithMigrateState` function and do it all in a single function always doing the migration

```hcl
func WithBackendConfig(backendConfig map[string]interface{}) tftOption {
	return func(f *TFBlueprintTest) {
		f.backendConfig = backendConfig
		f.migrateState = true
	}
}
```

Reply:
Yeah I think that makes sense, we can split it out if a usecase presents itself in the future.

Reply:
`WithMigrateState` merged  into `WithBackendConfig`